### PR TITLE
code-editor added

### PR DESCRIPTION
**Pull request by maradadivyasree020**
Closes: 1396

- **Title** : CodeEditor
- **Name:** : Marada Divya Sree
- **Idenitfy yourself:** : GSSOC

### Describe the add-ons or changes you've made 📃
I have added my project CodeEditor . This is an code editor application built with React.js. It allows users to write, edit, and execute code snippets in programming languages like HTML,CSS,JS. The editor provides syntax highlighting, and other essential features to enhance the coding experience.

### How Has This Been Tested? ⚙️

Yes, I have tested it on my local computer.

### Screenshots 📷

screenshot webp

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: React-JS-Projects/Intermediate/Code-Editor/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+3. Open the project folder in your preferred code editor, now you can view website in live.
+
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+<img src='./screenshot.webp.png'>
```
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Intermediate/Code-Editor/README.md:51]: after changing the ss update it here.

File: React-JS-Projects/Intermediate/Code-Editor/screenshot.webp.png
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Intermediate/Code-Editor/screenshot.webp.png:1]: `screenshot.webp` is not a name. Here `screenshot` is the name and `webp` is the extension of the ss file.
So, convert your png file into webp format and update it here.

File: React-JS-Projects/README.md
Comment (Avdhesh-Varshney) [inline React-JS-Projects/README.md:1]: I've assigned you `level3` to keep it under Advanced level project. Change the level from intermediate to advanced.

File: database/react.json [hunk 2]
```diff
@@ -33,5 +33,10 @@
 		"tag": "Intermediate",
 		"title": "Currency Converter",
 		"description": "Ths is a simple currency converter application that allows users to convert between different currencies. It fetches live exchange rates from an API and performs conversions accordingly."
+	},
+	{
+		"tag": "Intermediate",
```
Comment (Avdhesh-Varshney) [inline database/react.json:38]: change tag here also to advanced.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@maradadivyasree020 Looks good. But keep your project inside the `Advanced` projects directory as i've already assigned you `level3` ok.

File: React-JS-Projects/Intermediate/Code-Editor/README.md (hunk 1 above)
Comment (maradadivyasree020) [reply React-JS-Projects/Intermediate/Code-Editor/README.md:51]: sure

**Comment by maradadivyasree020** [comment]
I have made the required changes.

File: React-JS-Projects/Advanced/Code-Editor/screenshot.png
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Advanced/Code-Editor/screenshot.png:1]: convert this png to webp format and resolve the merge conflicts.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Review by Avdhesh-Varshney — APPROVED** [review]
@maradadivyasree020 PR Approved 🎉

**Comment by Avdhesh-Varshney** [comment]
@maradadivyasree020 mention your issue properly with the pr

**Pull request merged**